portal/src/main/webapp/js/src/pancancer_study_summary/pancancer_study_summary.js

```diff
@@ -468,16 +480,27 @@ var MinTotalSamplesSliderView = Backbone.View.extend({
             min: 0,
             max: this.max
         });
+        
+       this.$el.find(".diagram-general-slider-value").attr("id", "input2");
+         
     },
 
     // handle change to the slider
     handleSliderChange: function(e, ui) {
-        var sampleText = this.$el.find(".diagram-general-slider-value");
-        console.log("GENE: "+this.gene);
         // update text
-        sampleText.html(ui.value);
+        $("#input2").val(ui.value);
```

Comment:
Hi @JiaoJiao123 
In pan-cancer this won't work properly, as each gene subtab will have an input2 identifier now. If I change the slider for the second gene tab, it will update the value box in the first gene tab.